aspnetcore/fundamentals/configuration/platform-specific-configuration.md

```diff
@@ -63,7 +63,7 @@ Only part of the file is shown. The assembly name in the example is `StartupEnha
 
 ### Update the dependencies file
 
-The runtime location is specified in the *\*.deps.json* file. To active the enhancement, the `runtime` element must specify the location of the enhancement's runtime assembly. Prefix the `runtime` location with `lib/netcoreapp2.0/`:
+The runtime location is specified in the *\*.deps.json* file. To active the enhancement, the `runtime` element must specify the location of the enhancement's runtime assembly. Prefix the `runtime` location with `lib/netcoreapp2.1/`:
```

Comment:
I suggest making this more generic by replacing `netcoreapp2.1` with a placeholder like `<target_framework_moniker>`.